community/item-schema.json

```
@@ -580,30 +580,36 @@
 			"name_german": "Top 20 - Helden der interstellaren Streitkräfte",
 			"name_italian": "Top 20 - Eroi delle Forze Armate Interstellari",
 			"name_russian": "Один из лучших 20 - Герои межзвёздных вооружённых сил",
 			"name_schinese": "顶级前20 - 星际武装部队的英雄",
+			"briefing_name_brazilian": "HdFAI Top 20",
 			"briefing_name_english": "HoIAF Top 20",
 			"briefing_name_german": "HoIAF Top 20",
 			"briefing_name_italian": "Top 20 EdFAI",
 			"briefing_name_russian": "Один из лучших 20 ГМВС",
 			"briefing_name_schinese": "HoIAF赛季前20名",
+			"before_description_brazilian": "Finalizou a Temporada %season% na colocação #%rank% com a pontuação de %score%.",
 			"before_description_english": "Finished Season %season% in rank #%rank% with a score of %score%.",
 			"before_description_german": "Beende die Saison %season% auf Rang #%rank% mit einer Punktzahl von %score%.",
 			"before_description_italian": "Terminata la Stagione %season% con il rango #%rank% e un punteggio di %score%.",
 			"before_description_russian": "Завершил %season%-й сезон на позиции #%rank% с результатом: %score%.",
 			"before_description_schinese": "以 %score% 分的成绩完成了第 %season% 赛季，排名第 %rank% 。",
+			"description_brazilian": "Medalha concedida a aqueles que finalizaram a temporada dos Heróis das Forças Armadas Interestelares no top 20.",
 			"description_english": "Medal awarded to those who finished a season of Heroes of the Interstellar Armed Forces in the top 20.",
 			"description_german": "Medaille, die an diejenigen verliehen wird, die eine Saison von Heroes of the Interstellar Armed Forces in den Top 20 abgeschlossen haben.",
 			"description_italian": "Medaglia conferita a coloro che terminano una stagione di Eroi delle Forze Armate Interstellari nella top 20.",
 			"description_russian": "Медаль, выдаваемая тем, кто завершил сезон игры «Героев межзвёздных вооружённых сил» одним из лучших 20 участников.",
+			"after_description_brazilian": "Alienígenas Eliminados: %alien_kills%\nEliminações em Equipe: %player_kills%\nMissões Bem-Sucedidas: %games_won%\nMissões Falhadas: %games_lost%",
 			"after_description_english": "Aliens Killed: %alien_kills%\nTeam Kills: %player_kills%\nSuccessful Missions: %games_won%\nFailed Missions: %games_lost%",
 			"after_description_german": "Getötete Aliens: %alien_kills%\nTeam Kills: %player_kills%\nErfolgreiche Einsätze: %games_won%\nFehlgeschlagene Einsätze: %games_lost%\n",
 			"after_description_italian": "Alieni Uccisi: %alien_kills%\nUccisioni di Squadra: %player_kills%\nMissioni Completate: %games_won%\nMissioni Fallite: %games_lost%",
 			"after_description_russian": "Жуков убито: %alien_kills%\nСоюзников убито: %player_kills%\nМиссий выполнено: %games_won%\nМиссий провалено: %games_lost%",
-			"after_description_schinese": "击杀异形: %alien_kills%\n击杀队友: %player_kills%\n任务完成: %games_won%\n任务失败: %games_lost%",
+			"after_description_schinese": "击杀异形: %alien_kills%\n击杀队友: %player_kills%\n完成的任务: %games_won%\n失败的任务: %games_lost%",
```

Looks like an accidental rollback here
```suggestion
			"after_description_schinese": "击杀异形: %alien_kills%\n击杀队友: %player_kills%\n任务完成: %games_won%\n任务失败: %games_lost%",
```